test4.js

```diff
@@ -7,18 +7,23 @@ debugger;
 const getCartPrice = (cart, prices) => {
   let sum = 0;
   cart.forEach((prod) => {
-    sum = 0;
-    prices.forEach((prodpr) => {
-      // alert(Object.keys(prod)[0]);
-      // alert(Object.keys(prodpr)[0]);
-      if (Object.keys(prod)[0] == Object.keys(prodpr)[0]) {
-        sum = sum + Object.values(prod)[0] * Object.values(prodpr)[0];
-      }
-    });
-    alert(`fuuu ${prod} , sum ${sum}`);
-  });
+    pricc=prices.find(item=> Object.keys(item)[0]==Object.keys(prod)[0]);
+    sum=Object.values(pricc)[0]*Object.values(prod)[0];
```

Comment:
did you check your program for correctness?
it seems you override `sum` (needs to be `sum=sum + Object.values(pricc)[0]*Object.values(prod)[0];`)